OSSM-8862 Migration Hub page content (#88651)
openshift/openshift-docs: modules/ossm-migrating-hub-recommendations.adoc

// Module included in the following assemblies:
//
// * service-mesh-docs-main/migrating/ossm-migrating-from-2-to-3-assembly.adoc

:_mod-docs-content-type: REFERENCE
[id="ossm-migrating-hub-recommendations_{context}"]

> = Recommendations

Reviewer: Consider making this heading more specific. Maybe "Migration recommendations"?

> Given the nature of migrating to a brand new operator, the following recommendations are to help limit the risk of misconfigurations or possible conflicts:

Reviewer: ```suggestion
Consider the following recommendations to limit the risk of misconfigurations or possible conflicts:
```

> * If you have automated updates configured in the {ocp-product-title} web console, change it to **Manual**. This helps to prevent updates to the {SMProduct} 2 Operator and control planes during your migration.

Reviewer: ```suggestion
* If you have automated updates configured in the {ocp-product-title} web console, change them to **Manual**. By switching to manual updates, you can prevent updates to the {SMProduct} 2 Operator and control planes during your migration.
```

* Keep the amount of the service mesh configuration changes, such as changes for traffic management or security or adding new workloads or gateways, to a minimum during the migration.

[NOTE]
====

> If it's necessary to add a new workload namespace during the migration, it must be managed by the {SMProduct} 3 control plane and labeled with `maistra.io/ignore-namespace: "true"` to avoid conflicts between the {SMProduct} 3 control plane and the {SMProduct} 2 `ServiceMeshControlPlane` resource.

Reviewer: ```suggestion
If you need to add a new workload namespace during the migration, it must be managed by the {SMProduct} 3 control plane and labeled with `maistra.io/ignore-namespace: "true"` to avoid conflicts between the {SMProduct} 3 control plane and the {SMProduct} 2 `ServiceMeshControlPlane` resource.
```

// Module included in the following assemblies:
//
// * service-mesh-docs-main/migrating/ossm-migrating-from-2-to-3-assembly.adoc

:_mod-docs-content-type: REFERENCE
[id="ossm-migrating-hub-recommendations_{context}"]
= Recommendations

Given the nature of migrating to a brand new operator, the following are recommendations to help limit the risk of misconfigurations or possible conflicts:

* If you have automated updates configured in the {ocp-product-title} web console, change it to **Manual**. This helps to prevent updates to the {SMProduct} 2 Operator and control planes during your migration.

* Keep the amount of the service mesh configuration changes, such as changes for traffic management or security or adding new workloads or gateways, to a minimum during the migration.

[NOTE]
====
If it's necessary to add a new workload namespace during the migration, it must be managed by the {SMProduct} 3 control plane and labeled with `maistra.io/ignore-namespace: "true"` to avoid conflicts between the {SMProduct} 3 control plane and the {SMProduct} 2 `ServiceMeshControlPlane` resource.
====

* Finish the migration without unnecessary delays

> Once the migration is started, it is recommended to complete it as quickly as possible.

Reviewer 2: Move this to the previous bullet? Or put in a separate bullet?
